packages/calcite-components/src/components/time-picker/time-picker.e2e.ts

```diff
@@ -1214,7 +1214,7 @@ describe("calcite-time-picker", () => {
       await page.mouse.click(buttonUpLocationX, buttonUpLocationY);
       await page.waitForChanges();
       const fractionalSecondEl = await page.find(`calcite-time-picker >>> .input.fractionalSecond`);
-      expect(fractionalSecondEl.innerHTML).toEqual("0");
+      expect(fractionalSecondEl.textContent).toEqual("0");
```

Comment:
Lit inserts comments in the rendered content to help it deal with cases when nodes are inserted before/after lit-rendered nodes by third-party code.
To avoid those comments from appearing in this test, use textContent instead of innerHTML